src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/PyrisStatusUpdateService.java

```diff
@@ -17,20 +21,30 @@ public class PyrisStatusUpdateService {
 
     private final IrisChatSessionService irisChatSessionService;
 
+    private static final Logger log = LoggerFactory.getLogger(PyrisStatusUpdateService.class);
+
     public PyrisStatusUpdateService(PyrisJobService pyrisJobService, IrisChatSessionService irisChatSessionService) {
         this.pyrisJobService = pyrisJobService;
         this.irisChatSessionService = irisChatSessionService;
     }
 
     /**
-     * Handles the status update of a tutor chat job and forwards it to {@link IrisChatSessionService#handleStatusUpdate(TutorChatJob, PyrisTutorChatStatusUpdateDTO)}
+     * Handles the status update of a tutor chat job and forwards it to {@link IrisChatSessionService#handleStatusUpdate(TutorChatJob, PyrisStatusUpdateDTO)}
      *
      * @param job          the job that is updated
      * @param statusUpdate the status update
      */
-    public void handleStatusUpdate(TutorChatJob job, PyrisTutorChatStatusUpdateDTO statusUpdate) {
-        irisChatSessionService.handleStatusUpdate(job, statusUpdate);
-
+    public void handleStatusUpdate(PyrisJob job, PyrisStatusUpdateDTO statusUpdate) {
```

Comment:
Each job type should have its own method, as each job type will have its own DTO type